core/roboconf-target-docker/src/main/java/net/roboconf/target/docker/internal/DockerHandler.java

```diff
@@ -148,7 +148,7 @@ else if(( image = DockerUtils.findImageByTag( imageId, images )) != null )
 			.createContainerCmd(imageId)
 			.withCmd("/usr/local/roboconf-agent/start.sh",
 						"application-name=" + applicationName,
-						"root-instance-name=" + rootInstanceName,
+						"scoped-instance-path=" + scopedInstancePath,
```

Comment:
Probably something to fix in the Dockerfile generator as well (there is a script there). Can be done later...